pages/common/xetex.md

```diff
@@ -0,0 +1,16 @@
+# xetex
+
+> Compile a PDF document from XeTeX source files.
+> More information: <https://manned.org/xetex>.
+
+- Compile a PDF document:
+
+`xetex {{source.tex}}`
+
+- Compile a PDF document specifying an output directory:
+
+`xetex -output-directory={{path/to/directory}} {{source.tex}}`
+
+- Compile a PDF document, halting on each error:
```

Comment:
```suggestion
- Compile a PDF document, stopping on each error:
```

Does this exit or just highlight?

Reply:
It exits.

Reply:
Then here is a better alternative

```suggestion
- Compile a PDF document, exiting if errors occur:
```

Reply:
I think we should also apply this to `pdflatex`, `latex`, `pdftex` and `tex`.

Reply:
That would have been my next PR, once this is merged.